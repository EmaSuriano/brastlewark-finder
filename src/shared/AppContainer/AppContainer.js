import React from 'react';
import styled from 'styled-components';

const AppContainer = styled.div`
  background: lightblue;
  display: flex;
  height: 100vh;
  flex-direction: column;
  align-items: center;
`;

export default AppContainer;
