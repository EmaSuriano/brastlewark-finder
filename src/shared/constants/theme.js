export default {
  background: '#4F5D75',
  light: '#FFFFFF',
  disabled: '#BFC0C0',
  accent: '#EF8354',
  secondary: '#2D3142',
};
