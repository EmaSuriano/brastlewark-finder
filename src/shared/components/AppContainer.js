import styled from 'styled-components';
import { ThemePropTypes } from 'shared/constants/theme';

const AppContainer = styled.div`
  background: ${props => props.theme.background};
  height: 100vh;
  overflow-y: auto;
  min-width: 600px;
`;

AppContainer.propTypes = {
  theme: ThemePropTypes,
};

AppContainer.displayName = 'AppContainer';

export default AppContainer;
