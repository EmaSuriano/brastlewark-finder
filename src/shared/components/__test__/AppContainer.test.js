import React from 'react';
import { shallow } from 'enzyme';
import theme from 'shared/constants/theme';
import AppContainer from '../AppContainer';

describe('<AppContainer />', () => {
  it('render', () => {
    expect(shallow(<AppContainer theme={theme} />)).toMatchSnapshot();
  });
});
