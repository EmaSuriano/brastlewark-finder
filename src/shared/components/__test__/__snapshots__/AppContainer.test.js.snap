// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<AppContainer /> render 1`] = `
.c0 {
  background: #4F5D75;
  height: 100vh;
  overflow-y: auto;
  min-width: 600px;
}

<div
  className="c0"
/>
`;
